aggie-coding-club/Rev-Registration
# About us

## Description

This adds the about us credits dialog to the landing page.

## How to test

Click `About Rev Registration` on the landing page and make sure it's formatted correctly

## Screenshots

![chrome_QpWc2gOFVz](https://user-images.githubusercontent.com/7295783/104639457-4cc6a980-565c-11eb-8b26-9af1141c42c7.png)

![chrome_8dJLUZmyJN](https://user-images.githubusercontent.com/7295783/105221355-d8817f80-5b0d-11eb-8309-ce61f7ca00b5.png)

## Related tasks

Closes #174
